### PR TITLE
feat(AIWriter): Extract subject from response

### PR DESCRIPTION
**Pull request by valentinperignon**

### CODE REVIEW

File: MailCore/Utils/Constants.swift [hunk 1]
```diff
@@ -65,6 +65,8 @@ public enum Constants {
         try! NSRegularExpression(pattern: ">\\s*<|>?\\s+<?")
     }()
 
+    public static let aiRegex = "^[^:]+:(?<subject>.+?)\n\\s*(?<content>.+)"
```
Comment (adrien-coye) [inline MailCore/Utils/Constants.swift:68]: maybe name it `aiDetectSubjectRegex` ?

**Review by adrien-coye — APPROVED** [review]
We discussed my remarks. I just have a tiny comment about some minor detail. Good to merge when you want.

**Review by Lnamw — DISMISSED** [review]

File: MailCore/Utils/Constants.swift (hunk 1 above)
Comment (adrien-coye) [reply MailCore/Utils/Constants.swift]: all good thanks 👍

**Review by adrien-coye — DISMISSED** [review]
LGTM

**Review by Lnamw — APPROVED** [review]

**Pull request merged**